datasets/loader_classification.py

```diff
@@ -688,6 +722,51 @@ def skin_segmentation(dataset_dir: Path) -> bool:
     return True
 
 
+def cifar(dataset_dir: Path) -> bool:
```

Comment:
it seems that by the result of the function you will never get a `False` value...

Reply:
The loaders in this file follow the general look of the loaders in the sklearn bench and should return True as a success status. Otherwise it will result in errors here
https://github.com/IntelPython/scikit-learn_bench/blob/master/datasets/load_datasets.py#L78